src/main/flight/pid.c

```diff
@@ -538,6 +567,79 @@ static void handleItermRotation()
     }
 }
 
+#ifdef USE_ACRO_TRAINER
+
+int8_t acroTrainerSign(float x)
```

Comment:
Replace with `int`, there's no use in passing less than full word types as full words are passed and used for arithmetics anyway, it may actually be slower.